File: v2/android/src/main/java/io/swagger/client/model/SourceInput.java
package io.swagger.client.model;


import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class SourceInput  {
  
  @SerializedName("name")
  private String name = null;
  @SerializedName("url")
  private String url = null;
  @SerializedName("logo_url")
  private String logoUrl = null;
  @SerializedName("enabled")
  private Boolean enabled = null;
  @SerializedName("custom_rss_feed")
  private Boolean customRssFeed = null;
  @SerializedName("rss_publisher")
  private String rssPublisher = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  public String getName() {
    return name;
  }
  public void setName(String name) {
    this.name = name;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  public String getUrl() {
    return url;
  }
  public void setUrl(String url) {
    this.url = url;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getLogoUrl() {
    return logoUrl;
  }
  public void setLogoUrl(String logoUrl) {
    this.logoUrl = logoUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Boolean getEnabled() {
    return enabled;
  }
  public void setEnabled(Boolean enabled) {
    this.enabled = enabled;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Boolean getCustomRssFeed() {
    return customRssFeed;
  }
  public void setCustomRssFeed(Boolean customRssFeed) {
    this.customRssFeed = customRssFeed;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  public String getRssPublisher() {
    return rssPublisher;
  }
  public void setRssPublisher(String rssPublisher) {
    this.rssPublisher = rssPublisher;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SourceInput {\n");
    
    sb.append("  name: ").append(name).append("\n");
    sb.append("  url: ").append(url).append("\n");
    sb.append("  logoUrl: ").append(logoUrl).append("\n");
    sb.append("  enabled: ").append(enabled).append("\n");
    sb.append("  customRssFeed: ").append(customRssFeed).append("\n");
    sb.append("  rssPublisher: ").append(rssPublisher).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/SourceInput.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.Model {
  [DataContract]
  public class SourceInput {
    
    
    [DataMember(Name="name", EmitDefaultValue=false)]
    public string Name { get; set; }

    
    
    [DataMember(Name="url", EmitDefaultValue=false)]
    public string Url { get; set; }

    
    
    [DataMember(Name="logo_url", EmitDefaultValue=false)]
    public string LogoUrl { get; set; }

    
    
    [DataMember(Name="enabled", EmitDefaultValue=false)]
    public bool? Enabled { get; set; }

    
    
    [DataMember(Name="custom_rss_feed", EmitDefaultValue=false)]
    public bool? CustomRssFeed { get; set; }

    
    
    [DataMember(Name="rss_publisher", EmitDefaultValue=false)]
    public string RssPublisher { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SourceInput {\n");
      
      sb.Append("  Name: ").Append(Name).Append("\n");
      
      sb.Append("  Url: ").Append(Url).Append("\n");
      
      sb.Append("  LogoUrl: ").Append(LogoUrl).Append("\n");
      
      sb.Append("  Enabled: ").Append(Enabled).Append("\n");
      
      sb.Append("  CustomRssFeed: ").Append(CustomRssFeed).Append("\n");
      
      sb.Append("  RssPublisher: ").Append(RssPublisher).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
